SortingStation/sortingStation.c

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include "stack.c"
+#include <string.h>
+
+#define MAXSIZE 100
+
+bool isOperation(char element){
+    if (element == '+' || element == '-' || element == '*' || element == '/'){
+        return true;
+    }
+    return false;
+}
+
+bool isDigit(char element){
+    if (element == '0' || element == '1' || element == '2' || element == '3' || element == '4' || element == '5' || element == '6' || element == '7' || element == '8' || element == '9'){
+        return true;
+    }
+    return false;
+}
+
+int precedence(char element) {
+    if (element == '+' || element == '-'){
+         return 1;
+    }
+    else if (element == '*' || element == '/'){
+        return 2;
+    }
+    return 0;
+}
+
+void takeLasttwoElementsFromTheStack(Stack* stack, int* firstNumber, int* secondNumber){
+    *firstNumber = theLastElementOfStack(stack);
+    pop(stack);
+    *secondNumber = theLastElementOfStack(stack);
+    pop(stack);
+}
+
+void convertStringFromInfixToPostfixNotation(const char* stringToConvert, char* outputString){
```

Comment:
```suggestion
void convertStringFromInfixToPostfixNotation(const char* stringToConvert, char* outputString) {
```
Тут и везде, где применимо :)